### ISSUE TITLE
ipa install failed

### ISSUE THREAD

**Issue by zhumxiang** [risporce/Supercell-jailbreak#83]
I followed the steps to obtain the new COC executable file, but when I repackaged the IPA and attempted to install it using TrollStore, TrollStore reported an error "Install Error 180. The app you tried to install has an encrypted main binary...." I suspect that the new executable file is invalid.

**Comment by VuHoangLong2k1** [comment]
Plss share me ipa

**Comment by zhumxiang** [comment]
> https://github.com/rclean/Supercell-jailbreak-Clash-of-Clans here can download new version support TH17. thanks rispoce uploading ipa for more than 1 years @VuHoangLong2k1 @zhumxiang

It's great that there is new ipa provided! However the file you provided seems to be injected with cheat plugin. I know the file must be from China because I'm just a Chinese :), I'm worried about the security of my account so I won't use it. Thanks anyway.

**Comment by rclean** [comment]
Yes, I'm one of the users, have been using it for years.You can just ignore it. The game account is logged in with authorization from the Apple account, so there should be no security issues. That's why I'm sharing it. @zhumxiang

**Comment by VuHoangLong2k1** [comment]
@rclean omg thanks for sharing

I hope you keep that trust and update regularly. Respect bro ♥️

**Comment by Alzentane** [comment]
https://www.mediafire.com/file/my7f1a0fofu3ymt/CoCNJ.ipa/file
